docs/content/using/using_hostdefinition.md

```diff
@@ -0,0 +1,72 @@
+# Using dynamic host definition
+
+Dynamic host connectivity removes the necessity for manual host definitions. This is also facilitates the scaling out process of a node. 
+
+The `HostDefiner` identifies the hosts available for host definition on each storage system and controls each of the host definitions. To see the statuses of the managed host definitions by the host definer, use the `hostdefinition` command.
+
+Use the `hostdefinitions` command to see the phase status of all definitions on the storage side.
+
+     $> kubectl get hostdefinitions
+
+|Phase|Description|
+|---------|--------|
+|Ready|Host definition is created and is working as expected.|
+|PendingForCreation|Host definition did not complete during the last attempt. The `HostDefiner` will try again.|
+|PendingForDeletion|Host deletion did not complete during the last attempt. The `HostDefiner` will try again.|
+|Error|Host definition or deletion did not complete and will not try again.|
+
+## Recovering from an Error state
+
+If any of the host definitions have an Error status, follow this procedure to have the `HostDefiner` reattempt to define the hosts.
+
+1. Undeploy the CSI node pod from the relevant node that the `HostDefinition` is a part of.
+2. Verify that all `HostDefinition` instances of the node are deleted.
+     
+          $> kubectl get hostdefinition | grep <hostname> | wc -l
+     
+     The output should be `0`.
+3. Redeploy the CSI node pod on the relevant node.
+
+     The HostDefiner handles all of the new host definitions.
```

Comment:
```suggestion
     The host definer handles all of the new host definitions.
```

Reply:
updated

Reply:
I still see `HostDefiner` in many places.
IMO `HostDefiner` is the CR name, and the component name is simply `host definer` (similar to how the "CSI driver" CR is `IBMBlockCSI`)

Reply:
fixed

Reply:
the `HostDefiner` custom resource merely tells the operator to deploy the host definer.

so in places such as:
>The `HostDefiner` custom resource handles changes in the Kubernetes clusters that relate to the host definition feature

the CR should be replaced by the "host definer" itself

Reply:
now I see:
>The host definer custom resource can optionally be installed as part of this process.

but, as I wrote, the custom resource name is actually `HostDefiner`

Reply:
so every time that I write it together with custom resource, it should be written as `HostDefiner`?

Reply:
yes (except for places where it should be the custom resource, e.g. `handles changes in the Kubernetes clusters`)

Reply:
updated all instances and aligned with how IBMBlockCSI is written